tests/imgls.pl

```diff
@@ -166,35 +433,129 @@ sub get_image_bytes {
     return $filebytes;
 }
 
-sub get_cmd_output {
-    my $prog = shift;
+sub ls_sort {
+    return if ! (@_ or scalar @_ > 1);
 
-    use English;
-    my ($ret, $pid);
-    die "Can't fork: $!" unless defined($pid = open(KID, "-|"));
-    if ($pid) {           # parent
-        $ret .= $_       while <KID>;
-        close KID;
-        return $ret;
+    # XXX t and r issue and y
+    if ($opts{'t'}) {
+        # descending
+        @_ = sort { _lstat($b)->mtime <=> _lstat($a)->mtime } @_;
+    }
+    # ctime, then lexical within times
+    # does not match macOS' ls(1) cmd, which seems wrong (or at least does not
```

Comment:
It would be nice to sort this out; if it's too hard to get right it's better to omit the feature so people don't depend on behavior that changes subtlely later on

Reply:
macOS seems to sort lexically with -c/-U, but shows ctime timestamps.  So, I'm doing the same now.